### DISCUSSION TITLE
How do I connect kafka-ui to MSK managed Kafka in AWS?

### DISCUSSION THREAD

**Discussion by n1vgabay** [provectus/kafka-ui#3443] [Q&A] [answered]
Hi folks,

So quick short, I'm running the latest version of kafka-ui in eks cluster to msk kafka on aws.

I didn't understand at all how to configure the values to connect.

Other services have no issues.

We're using SASL plaintext auth with the msk service. 2 brokers in msk.

Any example, or direction will be glad helpful!
Thanks.

Niv

**Comment by Haarolean** [comment, marked as answer]
Hi, use [this example](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-sasl.yaml) as a start.
Let me know if you have any further questions.
